config.js

```diff
@@ -280,9 +340,9 @@ exports.updateFromArgv = function() {
         console.log("Warning: --maxactivegames argument has been renamed to --maxconnectedgamesperuser. Use --maxconnectedgamesperuser instead.");
     }
 
-    if (argv.botid || argv.bot || argv.id || argv.minrankedhandicap || argv.maxrankedhandicap || argv.minunrankedhandicap || argv.maxunrankedhandicap || argv.maxtotalgames || argv.maxactivegames) {
-        console.log("\n"); /*IF there is a warning, we skip a line to make it more pretty*/
-    }
+if (argv.botid || argv.bot || argv.id || argv.minrankedhandicap || argv.maxrankedhandicap || argv.minunrankedhandicap || argv.maxunrankedhandicap || argv.maxtotalgames || argv.maxactivegames || argv.maxmaintime || argv.maxmaintimeranked || argv.maxmaintimeunranked || argv.minmaintime || argv.minmaintimeranked || argv.minmaintimeunranked || argv.maxperiodtime || argv.maxperiodtimeranked || argv.maxperiodtimeunranked || argv.minperiodtime || argv.minperiodtimeranked || argv.minperiodtimeunranked) {
```

Comment:
Shouldn't there be 4 spaces before these line to match indent elsewhere?

Reply:
a PR without space review is not a PR after all @roy7 @Dorus :rofl: 

yes, this is because in my old PR #129 there was no 4 spaces at that part, it was added in #132 

will commit fix

Reply:
weird i thought i fixed it

Reply:
oh i guess i had planned to fix it until the "string()" issue popped, then forgot it lol

Reply:
commited :tada: